docs/access-management/rbac.md

```diff
@@ -0,0 +1,45 @@
+---
+products: oss-enterprise, cloud-teams
+---
+
+# Role Based Access Control (RBAC)
+
+Role Based Access Control allows a user with Administrative access to apply roles to users, granting different levels of permission within an Organization or Workspace. 
+
+RBAC roles do not require a customer to use SSO. These roles can be enabled on any Organization.
```

Comment:
lets please remove this sentence as it is misleading. SSO is required in Self-Managed to use RBAC, and most Teams customers will use RBAC alongside SSO as well